app/bot/openai/openai_test.go

```diff
@@ -82,7 +86,7 @@ func TestOpenAI_OnMessage(t *testing.T) {
 
 			assert.Equal(t,
 				tt.response,
-				o.OnMessage(Message{Text: fmt.Sprintf("chat! %s", tt.request), ID: 756}),
+				o.OnMessage(bot.Message{Text: fmt.Sprintf("chat! %s", tt.request), ID: 756}),
 			)
 			calls := mockOpenAIClient.CreateChatCompletionCalls()
 			assert.Equal(t, 1, len(calls))
```

Comment:
this one likely should be required not assert due to calls[0] below

Reply:
This is required for the case of 2+ calls. `calls[0]` wouldn't help in this case

Reply:
Not sure I follow. I mean required.Equal(t,1,len) because if it less (i.e. 0) the calls[0] will fail on panic for sure